dashboard/app/models/unit_group.rb

```diff
@@ -312,24 +309,19 @@ def self.has_any_course_experiments?(user)
     Experiment.any_enabled?(user: user, experiment_names: UnitGroupUnit.experiments)
   end
 
-  # Get the set of valid courses for the dropdown in our sections table.
-  def self.valid_courses_without_cache
```

Comment:
This does not seem to be used anywhere

Reply:
Thank you. it looks like this got orphaned in https://github.com/code-dot-org/code-dot-org/pull/42228.